lib/CL/pocl_cl.h

```diff
@@ -1217,6 +1227,30 @@ struct _cl_command_queue {
   void *data;
 };
 
+struct _cl_command_buffer_khr
+{
+  POCL_ICD_OBJECT;
+  POCL_OBJECT;
+
+  /* Queues that this command buffer uses */
+  cl_uint num_queues;
+  cl_command_queue *queues;
+
+  cl_uint num_properties;
+  cl_command_buffer_properties_khr *properties;
+
+  cl_command_buffer_state_khr state;
```

Comment:
Add comments to the member-variables which are likely not obvious to outsiders and you after at least one year of not touching the code.